deploy/ccm/envoy-config-collector-custom-proxy.yaml

```diff
@@ -121,6 +121,13 @@ static_resources:
             name: tls_sds
             sds_config:
               path: /etc/envoy/tls_certificate_sds_secret.yaml
+          validation_context:
+            trusted_ca:
+              filename: /etc/ssl/certs/ca-certificates.crt
+            match_typed_subject_alt_names:
+            - san_type: DNS
```

Comment:
some indentation issue here. need two more spaces and so are two following lines. 
this seems to apply to all "match_typed_subject_alt_names" section, please help verify again.
correct format should be like (https://github.dev.purestorage.com/parts/ccm-go/blob/master/deploy/envoy-config-rest.yaml#L77)

Reply:
it's just different yaml conventions, should be the same. san_type is sub component of match_typed_subject_alt_names

Reply:
cool, thanks for clarifying. I don't have any other issue now